### PR TITLE
Update homepage copy, portal title and favicon

### PR DESCRIPTION
**Pull request by axelboc**
image

image

### CODE REVIEW

**Review by noobadmin — APPROVED** [review]
Everything looks ok. Didn't find anything I'd change. Thanks you!

File: src/Home/HomePage.js [hunk 1]
```diff
@@ -17,30 +17,38 @@ function HomePage() {
...
             <FiSearch />
           </Button>
         </Flex>
-      </form>
+      </Box>
       <p>
-        The PaNOSC project brings together six strategic European research
-        infrastructures and two e-infrastructures:
+        The European Photon and Neutron sources are working together in the
+        PaNOSC and ExPaNDS projects financed by the European Commission to build
```
Comment (andygotz) [inline src/Home/HomePage.js:42]: please make PaNOSC and ExPaNDS links to the project home pages

File: src/Home/HomePage.js [hunk 2]
```diff
@@ -17,30 +17,38 @@ function HomePage() {
   return (
     <>
       <Heading as="h1" variant="display">
-        The Photon and Neutron Open Science Cloud
+        European Photon and Neutron Open Data Search Portal
       </Heading>
-      <form action="/search" onSubmit={handleSubmit}>
+      <Box as="form" action="/search" onSubmit={handleSubmit} sx={{ my: 4 }}>
+        <Text as="p" sx={{ mt: 0, mb: 2, fontStyle: 'italic' }}>
+          Type a query to search for open data from photon and neutron sources
```
Comment (andygotz) [inline src/Home/HomePage.js:24]: add an example 'e.g. data'

File: src/Home/HomePage.js [hunk 3]
```diff
@@ -17,30 +17,38 @@ function HomePage() {
...
-        The PaNOSC project brings together six strategic European research
-        infrastructures and two e-infrastructures:
+        The European Photon and Neutron sources are working together in the
+        PaNOSC and ExPaNDS projects financed by the European Commission to build
+        the <strong>European Open Science Cloud</strong>. One of the main
+        objectives of the EOSC is to make <strong>Open Data</strong> from these
+        facilities{' '}
+        <abbr title="Findable, Accessible, Interoperable, Reusable">FAIR</abbr>.
+        This portal implements the F(indable) part of FAIR via a{' '}
+        <strong>federated search engine</strong> from the following facilities:
```
Comment (andygotz) [inline src/Home/HomePage.js:48]: remove facilities which are not available yet

**Review by andygotz — COMMENTED** [review]
Some small comments from my side

**Pull request merged**